src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/GovUk.Education.ExploreEducationStatistics.Admin.Tests.csproj

```diff
@@ -35,6 +35,7 @@
 
   <ItemGroup>
     <Folder Include="Controllers\Api\UserManagement" />
+    <Folder Include="ViewModels" />
```

Comment:
I don't think you need this. I think it's been automatically generated in order to retain a folder that's empty (since `PublicationSaveViewModelTests` was removed) and would otherwise be deleted.

Reply:
I've removed the entries for both the ViewModels directory and the UserManagement directory.